jest-puppeteer.config.js

```diff
@@ -1,14 +1,49 @@
-const path = require('path')
-const extPath = path.join(__dirname, './build')
+const { join } = require('path')
+const { readdirSync, existsSync, lstatSync, unlinkSync, rmdirSync } = require('fs')
+
+// load envs
+require('dotenv').config({
+    path: join(__dirname, `./.env/e2e-${process.env.NODE_ENV}`),
```

Comment:
```suggestion
    path: join(__dirname, `./.env/e2e-${process.env.NODE_ENV || 'template'}`),
```
allow direct start, `yarn test:e2e`

Reply:
`NODE_ENV` won't be `undefined`.